core/server/api_container/server/service_network/default_service_network.go

```diff
@@ -1446,15 +1446,15 @@ func (network *DefaultServiceNetwork) destroyService(ctx context.Context, servic
 	}
 	successfullyDestroyedUuids, failedToDestroyUuids, err := network.kurtosisBackend.DestroyUserServices(context.Background(), network.enclaveUuid, userServiceFilters)
 	if err != nil {
-		return stacktrace.Propagate(err, "Attempted to destroy the services with UUID '%v' but had no success. You must manually destroy the services. Kurtosis will now try to remove its sidecar if it exists but might it fail as well.", serviceUuid)
+		return stacktrace.Propagate(err, "Attempted to destroy the service with UUID '%v' but had no success. You must manually destroy the service as well as its sidecar.", serviceUuid)
```

Comment:
are we sure that the service has a sidecar? (here and there)

if not perhaps

s/as its sidecar/as its sidecar if any/g